src/compiler/diagnosticMessages.json

```diff
@@ -2493,6 +2493,10 @@
         "category": "Error",
         "code": 2734
     },
+    "There is a 'this' type shadowed by this container.": {

```

Comment:
Suggestion: This container might have shadowed the value of 'this' that you meant to use.

Reply:
or: An outer value of 'this' is shadowed by this container.